vpr/src/base/ShowSetup.cpp

```diff
@@ -30,7 +30,11 @@ void ShowSetup(const t_vpr_setup& vpr_setup) {
     VTR_LOG("Circuit placement file: %s\n", vpr_setup.FileNameOpts.PlaceFile.c_str());
     VTR_LOG("Circuit routing file: %s\n", vpr_setup.FileNameOpts.RouteFile.c_str());
     VTR_LOG("Circuit SDC file: %s\n", vpr_setup.Timing.SDCFile.c_str());
-    VTR_LOG("Vpr floorplanning constraints file: %s\n", vpr_setup.FileNameOpts.read_vpr_constraints_file.c_str());
+    if (vpr_setup.FileNameOpts.read_vpr_constraints_file == "") {
+        VTR_LOG("Vpr floorplanning constraints file: not specified %s\n");
```

Comment:
This line has a bug that is crashing when run through the sanitizer regtests.  You have a format code (%s) with no matching argument, which can cause a seg fault.  Please fix and CI will relaunch.